umh.docs.umh.app/content/en/docs/features/connectivity/introduction-asset-management.md

```diff
@@ -51,4 +51,5 @@ This approach ensures a flexible, adaptable IIoT environment that addresses both
 
 ## Asset Management
```

Comment:
1. The page should be send to draft=false
2. Can we add more information from the upcoming user journey about asset management? https://github.com/united-manufacturing-hub/MgmtIssues/issues/1001